.github/workflows/android-ci.yml

```diff
@@ -70,6 +70,9 @@ jobs:
           distribution: "temurin"
           java-version: "17"
 
+      - name: Add aarch64-linux-android for Rust toolchian
+        run: rustup target add --toolchain stable-x86_64-unknown-linux-gnu aarch64-linux-android armv7-linux-androideabi i686-linux-android
```

Comment:
The dependency to rustup is not hight right now, so this is fine. However, we should make sure not to depend too much on rustup. This is a mistake that I made in maplibre-rs. The mistake later shows up e.g. when integrating into other build systems.

Ideally, the only required dependency is `rustc`. Realstically there is no way around `cargo`. However, a strict requirement for `rustup` can be easily avoided.

Using rustup in the CI is totalyl fine! (always!) We should just avoid depending on rustup in e.g. cmake scripts.

Reply:
thx, wasn't aware that rustup is a problem in cmake/bazel, so will make sure to avoid it there

Reply:
Not neccassarily a problem, but rustup is just one way of installing Rust.

Note that the rust-toolchain.toml is for example a feature of rustup. It might just cause frictioin to depend on rustup early on :)

Reply:
heh I love that you mention `rust-toolchain.toml` @maxammann ;) I think that this is one of the most underrated features for cross-platform development Ind I do use it in my projects. That said, your concern is completely correct in the case of MapLibre and other projects which use Bazel for build orchestration.